README.md

```diff
@@ -6,4 +6,25 @@
 
 *Welcome to the Red Hat Observability Group Handbook*
 
-> In progress.
+## Goals
+
+For now check the [Handbook Proposal](content/Proposals/Accepted/202106-handbook.md)
+
+## Using
+
+This handbook was designed to be used in two ways:
+
+* Through our website `https://rhobs-handbook.netlify.app/`.
+* Directly from GitHub UI by navigating to [`content`](https://github.com/rhobs/handbook/tree/main/content) directory.
```

Comment:
What about Github pages?

Reply:
Do you mean instead of hugo + netlify? We have some amazing pre-processors which makes GitHub UI and website working smoothly (you write docs like for GH UI) (: GH pages would lack of that.

Also we have rdy tooling as other website use it, so we can maintain one way of generating website.

Reply:
> We have some amazing pre-processors which makes GitHub UI and website working smoothly

Interesting, could you send me some links?

Reply:
* https://github.com/bwplotka/mdox/blob/main/main.go#L206
* https://github.com/rhobs/handbook/blob/f1860c7564b0f84192c26302aebfb3c3bbd49fc8/.mdox.yaml#L11